data/iconbrowser.appdata.xml.in

```diff
@@ -59,10 +59,10 @@
     <release version="1.2.0" date="2020-07-29">
       <description>
         <ul>
-          <li>Add audio-headphones and audio-headset</li>
-          <li>Add distribute actions</li>
+          <li>Add "audio-headphones" and "audio-headset"</li>
+          <li>Add GitHub Actions for distribution</li>
```

Comment:
Ah so this is actually about the `distribute-*` action icons. I'm not sure what would help make that most clear but here's a screenshot of what it is:

![Screenshot from 2022-09-27 14 03 47](https://user-images.githubusercontent.com/7277719/192634976-984b6013-68de-4348-820e-0fd4d9514445.png)

Reply:
Ah, I thought this means commit 83c878385dd938f9a065182a5bc14e4397412fd0.
What about `Add "distribute-*" action icons` then? :thinking:

Reply:
Yeah that works for me 👍